typescript/sdk/src/consts/multisigIsm.ts

```diff
@@ -213,6 +230,11 @@ export const defaultMultisigConfigs: ChainMap<MultisigConfig> = {
     ],
   },
 
+  osmosis: {
+    threshold: 1,
+    validators: ['0xea483af11c19fa41b16c31d1534c2a486a92bcac'],
```

Comment:
realizing I think we need Mitosis's validators here as well. not sure if Mitosis is using our default ISM or not, hopefully they're using their own ISM so this isn't something we need to go through multisig txs again for

Reply:
following up on discord about this

Reply:
probably when we get them - but same as before would wait til in flight fixes are merged and we get a fresh build